Build error
```
...
       >   COMPILING BINDINGS: ["src/bindings.cpp", "src/gl.cpp", "src/gpu.cpp", "src/shaper.cpp", "src/paragraph.cpp"]
       >     DEFINES: NDEBUG= SK_CODEC_DECODES_BMP= SK_CODEC_DECODES_WBMP= SK_R32_SHIFT=16 SK_ENABLE_PRECOMPILE= SK_GANESH= SK_DISABLE_TRACING= SK_USE_PERFETTO= SK_GAMMA_APPLY_TO_A8= SK_ENABLE_AVX512_OPTS= SKIA_IMPLEMENTATION=1 SK_TYPEFACE_FACTORY_FREETYPE= SK_FONTMGR_ANDROID_AVAILABLE= SK_FONTMGR_FREETYPE_DIRECTORY_AVAILABLE= SK_FONTMGR_FREETYPE_EMBEDDED_AVAILABLE= SK_FONTMGR_FREETYPE_EMPTY_AVAILABLE= SK_FONTMGR_FONTCONFIG_AVAILABLE= SK_GL= SK_SUPPORT_PDF= SK_CODEC_DECODES_JPEG= SK_CODEC_DECODES_ICO= SK_CODEC_DECODES_PNG= SK_HAS_WUFFS_LIBRARY= SK_CODEC_DECODES_GIF= SK_XML= SKSHAPER_IMPLEMENTATION=1 SK_SHAPER_PRIMITIVE_AVAILABLE= SK_SHAPER_HARFBUZZ_AVAILABLE= SK_SHAPER_UNICODE_AVAILABLE= SK_UNICODE_AVAILABLE= SK_UNICODE_ICU_IMPLEMENTATION= SK_ENABLE_PARAGRAPH= SKUNICODE_IMPLEMENTATION=1 U_SHOW_CPLUSPLUS_API=0 U_USING_ICU_NAMESPACE=0 U_DISABLE_RENAMING=1 SK_USING_THIRD_PARTY_ICU= U_COMMON_IMPLEMENTATION= U_STATIC_IMPLEMENTATION= U_ENABLE_DYLOAD=0 U_I18N_IMPLEMENTATION= _XOPEN_SOURCE=0
       >     ARGS: -std=c++17 -fno-rtti
       >   cargo:warning=src/bindings.cpp:81:10: fatal error: include/docs/SkPDFJpegHelpers.h: No such file or directory
       >   cargo:warning=   81 | #include "include/docs/SkPDFJpegHelpers.h"
       >   cargo:warning=      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       >   cargo:warning=compilation terminated.
       >
       >   --- stderr
       >
       >
       >   error occurred in cc-rs: command did not execute successfully (status code exit status: 1): LC_ALL="C" "/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "/nix/store/cngd1vy2y4ilbglnn84sm8k1wx9n2h4l-source" "-Wall" "-Wextra" "-std=c++17" "-fno-rtti" "-DNDEBUG=" "-DSK_CODEC_DECODES_BMP=" "-DSK_CODEC_DECODES_WBMP=" "-DSK_R32_SHIFT=16" "-DSK_ENABLE_PRECOMPILE=" "-DSK_GANESH=" "-DSK_DISABLE_TRACING=" "-DSK_USE_PERFETTO=" "-DSK_GAMMA_APPLY_TO_A8=" "-DSK_ENABLE_AVX512_OPTS=" "-DSKIA_IMPLEMENTATION=1" "-DSK_TYPEFACE_FACTORY_FREETYPE=" "-DSK_FONTMGR_ANDROID_AVAILABLE=" "-DSK_FONTMGR_FREETYPE_DIRECTORY_AVAILABLE=" "-DSK_FONTMGR_FREETYPE_EMBEDDED_AVAILABLE=" "-DSK_FONTMGR_FREETYPE_EMPTY_AVAILABLE=" "-DSK_FONTMGR_FONTCONFIG_AVAILABLE=" "-DSK_GL=" "-DSK_SUPPORT_PDF=" "-DSK_CODEC_DECODES_JPEG=" "-DSK_CODEC_DECODES_ICO=" "-DSK_CODEC_DECODES_PNG=" "-DSK_HAS_WUFFS_LIBRARY=" "-DSK_CODEC_DECODES_GIF=" "-DSK_XML=" "-DSKSHAPER_IMPLEMENTATION=1" "-DSK_SHAPER_PRIMITIVE_AVAILABLE=" "-DSK_SHAPER_HARFBUZZ_AVAILABLE=" "-DSK_SHAPER_UNICODE_AVAILABLE=" "-DSK_UNICODE_AVAILABLE=" "-DSK_UNICODE_ICU_IMPLEMENTATION=" "-DSK_ENABLE_PARAGRAPH=" "-DSKUNICODE_IMPLEMENTATION=1" "-DU_SHOW_CPLUSPLUS_API=0" "-DU_USING_ICU_NAMESPACE=0" "-DU_DISABLE_RENAMING=1" "-DSK_USING_THIRD_PARTY_ICU=" "-DU_COMMON_IMPLEMENTATION=" "-DU_STATIC_IMPLEMENTATION=" "-DU_ENABLE_DYLOAD=0" "-DU_I18N_IMPLEMENTATION=" "-D_XOPEN_SOURCE=0" "-o" "/build/source/target/x86_64-unknown-linux-gnu/release/build/skia-bindings-082a046e395d8240/out/skia/0602fb52cb66f316-bindings.o" "-c" "src/bindings.cpp"
       >
       >
       For full logs, run:
         nix log /nix/store/nyjhb1nzh3scvx6lwrk5i944g61civjr-neovide-0.15.0.drv
```

--- a/pkgs/by-name/ne/neovide/package.nix
+++ b/pkgs/by-name/ne/neovide/package.nix
@@ -26,17 +26,17 @@
 
 rustPlatform.buildRustPackage.override { stdenv = clangStdenv; } rec {
   pname = "neovide";
-  version = "0.14.1";
+  version = "0.15.0";
 
   src = fetchFromGitHub {
     owner = "neovide";
     repo = "neovide";
     tag = version;
-    hash = "sha256-tXKTKE2JrDDJDpnipCv1hk7vS/0i7nrjzqMoMAy53qM=";
+    hash = "sha256-MLiLddF53OXDPYuJbTAscezxN09mxZkuSOZtQz07JSE=";
   };
 
   useFetchCargoVendor = true;
-  cargoHash = "sha256-RKgpgM6quN0plYYE8fien5tUtHlN0YG0VTp5BGuzzuo=";
+  cargoHash = "sha256-1ni8AZIwAz5R2Ejt9Fj5qmybvL4KZV/M3BMqQx4HFLU=";
 
   SKIA_SOURCE_DIR =
     let
@@ -44,8 +44,8 @@ rustPlatform.buildRustPackage.override { stdenv = clangStdenv; } rec {
         owner = "rust-skia";
         repo = "skia";
         # see rust-skia:skia-bindings/Cargo.toml#package.metadata skia
-        tag = "m132-0.81.0";
-        hash = "sha256-9DQgCaCiK0zgsl0wARPEiGjyXxmNLRSYaHf3690KuCM=";
+        tag = "m135-0.83.1";
+        hash = "sha256-TSGPJl9DfWQtrkNIhv40s8VcuudCjbiSh+QjLc0hKN4=";
       };
       # The externals for skia are taken from skia/DEPS
       externals = linkFarm "skia-externals" (

